openapi.yaml

```diff
@@ -4353,9 +4353,15 @@ components:
         'openeo:property_filters':
           description: |-
             Gives a list of properties you can filter on when loading data from the collection.
+            Returning an empty array means that no filters are available.
+
+            This is a guide for users to get an overview over available filters.
```

Comment:
"available filters" ->  "available properties to filter on"

Reply:
```suggestion
            This is a guide for users to get an overview of available properties to filter on.
```